src/test/java/teammates/sqllogic/core/AccountsLogicTest.java

```diff
@@ -56,19 +57,21 @@ public void setUpMethod() {
     }
 
     @Test
-    public void testDeleteAccount_accountExists_success() {
+    public void testDeleteAccount_accountExists_success()
+            throws InvalidParametersException, EntityAlreadyExistsException {
         Account account = generateTypicalAccount();
         String googleId = account.getGoogleId();
 
-        when(accountsLogic.getAccountForGoogleId(googleId)).thenReturn(account);
-
+        accountsLogic.createAccount(account);
```

Comment:
Why is this method called at all? This is a unit test for delete. We do not need to create anything.

Reply:
In fact, accountsLogic.getAccountForGoogleId(googleId) will return null here. createAccount is similarly a no-op.

Reply:
> Why is this method called at all? This is a unit test for delete. We do not need to create anything.

I thought we'd need an account to delete. I have made the change and I think we are back to the existing TC from DeleteAccountAction.

> In fact, accountsLogic.getAccountForGoogleId(googleId) will return null here. createAccount is similarly a no-op.

You're right